src/agent/rtm.ts

```diff
@@ -30,6 +30,13 @@ import type {
 import { ChatAccess, Event, Properties, Push, RoutingStatus, RTMAPIOptions } from "../objects";
 import { RTMAPI } from "../internal/index";
 
+// internal interface to not allow subscribing for pushes from different version
+interface LoginRequestInternal extends LoginRequest {
+  pushes?: {
+    [key: string]: Pushes[];
+  };
+}
+
```

Comment:
i guess we do not need it anymore

Reply:
yes, I removed it only in customer, forgot to do it here